src/bootstrap/compile.rs

```diff
@@ -29,10 +29,11 @@ use crate::util::{exe, is_debug_info, is_dylib, output, symlink_dir, t, up_to_da
 use crate::LLVM_TOOLS;
 use crate::{CLang, Compiler, DependencyType, GitRepo, Mode};
 
-#[derive(Debug, PartialOrd, Ord, Copy, Clone, PartialEq, Eq, Hash)]
+#[derive(Debug, PartialOrd, Ord, Clone, PartialEq, Eq, Hash)]
 pub struct Std {
     pub target: TargetSelection,
     pub compiler: Compiler,
+    pub tail_args: Vec<String>,
```

Comment:
Am I reading this right -- only ever used with `-p=$crate`? If so, then I think it probably makes sense to just use a `Option<Interned<String>>` here as an explicit crate field, rather than going with tail_args and needing to drop Copy bounds and be more general than necessary.

It might also be worth adding an explicit constructor to Std (builder.std or Std::new) so that we can avoid awkwardly specifying None for the krate field all over the place (or tail_args).